demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -937,6 +948,23 @@ def _collect_mappers_dependencies(pack_mappers: list, id_set: dict, verbose: boo
                     packs_found_from_incident_fields)
                 mapper_dependencies.update(pack_dependencies_data)
 
+            if mapper_data.get('definitionId'):
+                packs_found_from_generic_types = PackDependencies._search_packs_by_items_names(
+                    related_types, id_set['GenericTypes'], exclude_ignored_dependencies)
+
+                if packs_found_from_generic_types:
+                    pack_dependencies_data = PackDependencies._label_as_mandatory(
+                        packs_found_from_generic_types)
+                    mapper_dependencies.update(pack_dependencies_data)
+
+                packs_found_from_generic_fields = PackDependencies._search_packs_by_items_names(
+                    related_types, id_set['GenericFields'], exclude_ignored_dependencies)
+
+                if packs_found_from_generic_fields:
+                    pack_dependencies_data = PackDependencies._label_as_mandatory(
+                        packs_found_from_generic_fields)
+                    mapper_dependencies.update(pack_dependencies_data)
```

Comment:
same comment as with classifiers - we do not want to check both generics and incident related fields and types.

Reply:
Done